src/components/Image.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import * as TYPES from "../store/actions";
+import {connect} from 'react-redux';
 
-const Image = ({image: {src, name}, onDoubleClick}) =>
+export const Image = ({image, updatePromptImages}) =>
   <img
-    src={src}
-    alt={name}
-    onDoubleClick={onDoubleClick}
+    src={image.src}
+    alt={image.name}
+    onDoubleClick={() => updatePromptImages(image.id)}
   />
 
-export default Image;
+export const mapDispatchToProps = (dispatch) => ({
```

Comment:
it doesn't look like this is being tested

Reply:
Yes, I have added this test